src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -72,11 +73,23 @@
            [quo/text (subs literal 0 (dec (count literal)))]])
     blocks))
 
+(defn add-edited-tag [parsed-text]
```

Comment:
btw just realized this should be done on the go side

Reply:
I was chatting with Pedro yesterday about the performance drawbacks of adding the `(Edited)` appended to the last piece of text rather than at a fixed location on the bottom right for example like this
![image](https://user-images.githubusercontent.com/33176106/222394018-88239bbc-8ca4-450d-8345-1ea7b3c64b7b.png)
But I did what I can to do the proposed design .. But you're right .. Calculations should be on `status-go` but the thing is this will force status-desktop to have the same message body for showing `(Edited)` status and they'll have to change how they did .. I think they did on the client-side
WDYT ?

Reply:
yes thats fine

Reply:
What do you think about this @cammellos?

Reply:
I am a bit uneasy about changing `parsedText`, since `parsedText` contains text that's generated from the user, and here we are appending something that is not user generated but metadata.

Having said that, I don't see any other cleaner solution (I thought about using the reduce step instead of modifying, but the result is not much better).

Whether to move it in status-go, I would delay the decision. `ParsedText` is stored in the database, and if we add `Edited` to it, it means we can't easily go back, since migrating it would be really annoying. 
We could generate it on fly when serializing the message, that's ok, but I'd hold on for now until we are sure this is the right approach. So I would leave a comment saying precisely that, that we should move it to status-go once we are sure this is how both mobile/desktop wants it to be.

As a side note `add-editet-tag` does not seem to take into consideration code blocks/quotes, since if the message ends with one of those, it does not use `children`.

Try with a message such as:

```
this is a code block
```

Reply:
Added the case for codeblocks/blockquotes
![image](https://user-images.githubusercontent.com/33176106/222415547-a94dc70c-7e80-4dad-a010-36072e03d428.png)

Reply:
@flexsurfer kindly take another look